sphinx_gallery/notebook.py

```diff
@@ -171,7 +183,17 @@ def fill_notebook(work_notebook, script_blocks):
         if blabel == 'code':
             add_code_cell(work_notebook, bcontent)
         else:
-            add_markdown_cell(work_notebook, bcontent + '\n')
+            try:
+                import pypandoc  # noqa
+            except ImportError:
+                logger.warning("pypandoc not available. Using Sphinx-Gallery "
```

Comment:
I think we'd want to check for a user option here to explicitly use pypandoc, else we'll start throwing warnings to users all over the place

Reply:
Yeah I realised this and have made those changes. Come back in 5! :)

Reply:
Also putting the warning in a for loop was not a good idea!

Reply:
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?
What do you mean not a good idea?